lom/src/lib/lomcommon/config.go

```diff
@@ -658,3 +659,51 @@ func InitConfigPath(path string) error {
     _, err := InitConfigMgr(cfgFiles)
     return err
 }
+
+/* Gets settings of int type from json string if present. Else returns defaultValue */
+func GetIntConfigurationFromJson(jsonString string, configurationKey string, defaultValue int) int {
+	if jsonString == "" {
```

Comment:
Use named return variable and you can init that value at the start of the function. Any return statement will return that value.

Reply:
I will keep it as it is for now.